data-raw/save_adlb.R

```diff
@@ -1,10 +1,15 @@
 # test file from
 # https://raw.githubusercontent.com/RhoInc/data-library/master/data/clinical-trials/renderer-specific/adbds.csv"
+# q: how to convert micro mol/L to mg/dL for creatinine?
 
-adlb <- read.csv("https://raw.githubusercontent.com/RhoInc/data-library/master/data/clinical-trials/renderer-specific/adbds.csv",
+
+adlb_ <- read.csv("https://raw.githubusercontent.com/RhoInc/data-library/master/data/clinical-trials/renderer-specific/adbds.csv",
                  encoding = "UTF-8") %>%
-  mutate(STRESN  = ifelse(TEST == "Creatinine" & STRESU == "μmol/L", STRESN * .0113, STRESN), #Convert μmol/L to mg/dL
-         STRESU  = ifelse(TEST == "Creatinine" & STRESU == "μmol/L", "mg/dL", STRESU),
+  mutate(STRESN  = ifelse(TEST == "Creatinine" & STRESU == "μmol/L", STRESN / 88.4, # 1 mg/dL creatinine = 88.4 µmol/L
+                          ifelse(TEST == "Blood Urea Nitrogen" & STRESU == "mmol/L", STRESN * 2.86, STRESN) # 1 mmol/L BUN = 2.8 mg/dL BUN
+                          ), # Convert μmol/L to mg/dL and mmol/L to mg/dL
+         STRESU  = ifelse(TEST %in% c("Creatinine", "Blood Urea Nitrogen", "Albumin"), "mg/dL", 
```

Comment:
wouldn't we need the albumin conversion to mg/dL you have in the app example code here if we're changing it's unit to mg/dL?

Reply:
Thanks - it's been updated. To be consistent with the ISG demo app.